roxctl/scanner/uploaddb/uploaddb_test.go

```diff
@@ -0,0 +1,22 @@
+package uploaddb
+
+import (
+	"io/fs"
+	"testing"
+
+	"github.com/pkg/errors"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestScannerUploadDbCommandFail(t *testing.T) {
+	t.Run("non existing filename", func(t *testing.T) {
```

Comment:
nit: Why the `t.Run` within a test with only a single test case? Currently, we don't follow this pattern throughout the code base.

Reply:
Good point! ✅